spec/spec_helper.rb

```diff
@@ -22,12 +22,33 @@
 
 Magma.instance.configure(YAML.load(File.read('config.yml')))
 
+Magma.instance.setup_db
+
+def truncate_attributes_table_on_error(&block)
+  yield
+rescue => error
+  Magma.instance.db[:attributes].truncate
+  raise error
+end
+
+truncate_attributes_table_on_error do
+  Magma.instance.db.run(<<~SQL
```

Comment:
This is interesting - given that we might *often* want to do such a thing, even in production - should this become a command?

Reply:
Yeah, that's an option. I guess the command would take a file path as an argument?